dali/test/python/test_operator_constant.py

```diff
@@ -12,76 +12,82 @@
-        self.const6 = ops.Constant(device=device, idata=-4, shape=(10,20))
+        super().__init__(10, 3, device_id=0, exec_async=True, exec_pipelined=True)
+        self.const1 = ops.Constant(device=device, fdata=(1.25, 2.5, 3))
+        self.const2 = ops.Constant(device=device, idata=(1, 2, 3, 4), shape=(2, 1, 2))
+        self.const3 = ops.Constant(device=device, idata=(-1, 1, 2, 3, 4), dtype=types.UINT8)
+        self.const4 = ops.Constant(device=device, fdata=(0.25, 1.25, 2.25, 3.25, 4.25),
+                                   dtype=types.FLOAT16)
+        self.const5 = ops.Constant(device=device, fdata=5.5, shape=(100, 100))
+        self.const6 = ops.Constant(device=device, idata=-4, shape=(10, 20))
         self.const7 = ops.Constant(device=device, idata=[0, 1, 0], dtype=types.BOOL)
 
-
     def define_graph(self):
-        return (self.const1(), self.const2(), self.const3(), self.const4(),
-                self.const5(), self.const6(), self.const7())
+        return self.const1(), self.const2(), self.const3(), self.const4(), self.const5(), \
+               self.const6(), self.const7()
 
 
 class ConstantFnPipeline(Pipeline):
     def __init__(self, device, array_interface):
-        super(ConstantFnPipeline, self).__init__(10, 3, device_id=0, exec_async=True, exec_pipelined=True)
+        super().__init__(10, 3, device_id=0, exec_async=True, exec_pipelined=True)
         self.device = device
         self.array = array_interface[0]
         self.dtype = array_interface[1]
         if self.dtype is None:
-            self.dtype = lambda x : x
+            self.dtype = lambda x: x
 
     def define_graph(self):
         device = self.device
         return [
-            types.Constant(device=device, value=(1.25,2.5,3)),
-            types.Constant(device=device, value=self.array([[[1,2]],[[3,4]]], dtype=self.dtype('int32'))),
-            types.Constant(device=device, value=self.array([0,1,2,3,4], dtype=self.dtype('uint8'))),
-            types.Constant(device=device, value=self.array([0.25,1.25,2.25,3.25,4.25], dtype=self.dtype('float16'))),
-            types.Constant(device=device, value=5.5, shape=(100,100), name="large"),
-            types.Constant(device=device, value=-4, shape=(10,20)),
+            types.Constant(device=device, value=(1.25, 2.5, 3)),
+            types.Constant(device=device,
+                           value=self.array([[[1, 2]], [[3, 4]]], dtype=self.dtype('int32'))),
+            types.Constant(device=device,
+                           value=self.array([0, 1, 2, 3, 4], dtype=self.dtype('uint8'))),
+            types.Constant(device=device, value=self.array([0.25, 1.25, 2.25, 3.25, 4.25],
+                                                           dtype=self.dtype('float16'))),
+            types.Constant(device=device, value=5.5, shape=(100, 100), name="large"),
+            types.Constant(device=device, value=-4, shape=(10, 20)),
             types.Constant(device=device, value=[False, True, False])
```

Comment:
(if following the previous suggestion)
```suggestion
            types.Constant(device=device,
                           value=[False, True, False])
```

Reply:
Done